lib/dav/dav.ts

```diff
@@ -12,10 +12,10 @@ import { NodeData } from '../files/nodeData'
 import { parsePermissions } from './davPermissions'
 import { getFavoritesReport } from './davProperties'
 
-import { getCurrentUser, getRequestToken, onRequestTokenUpdate } from '@nextcloud/auth'
-import { generateRemoteUrl } from '@nextcloud/router'
 import { CancelablePromise } from 'cancelable-promise'
 import { createClient, getPatcher } from 'webdav'
+import { generateRemoteUrl } from '@nextcloud/router'
```

Comment:
Why sorting down? `@...` should be before c...

Reply:
I sort the lines, not the libraries, have been doing this everywhere here for... Years 😂
Funny you're only noticing this now 🤭